<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>658</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>996</width>
    <height>658</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>996</width>
    <height>658</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Visualizza Cliente</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>658</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>VisualizzaCliente2.png</pixmap>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>638</x>
     <y>329</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nome">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>160</y>
     <width>197</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cognome">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>200</y>
     <width>197</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="codFiscale">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>241</y>
     <width>197</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>285</y>
     <width>197</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataNascita">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>325</y>
     <width>195</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="eta">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>365</y>
     <width>197</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefono">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>407</y>
     <width>197</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>450</y>
     <width>197</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataInizio">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataScadenza">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>437</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey;
color: black;
border-radius: 10px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="VisualizzaCliente2.qrc"/>
 </resources>
 <connections/>
</ui>
